### PR TITLE
add link to CrowdIn in README.md

### PR DESCRIPTION
**Pull request by AzariahAeroborn**
Start with a simple change, and then we can figure out if we want to add something to the i18n docs page or otherwise.

### CODE REVIEW

**Review by KelosPelmand — DISMISSED** [review]
Looks good to me.  Most I might suggest would be some sort of "we only do officially supported languages", but the people that statement would target probably come straight to discord, so they wouldn't read the readme file.

**Comment by AzariahAeroborn** [comment]
> I'm with Kelos, would be good to have a note about supported language and also have a list of what those languages are, prob with the English name, localised name, locale name, and maybe a flag because it's cool.

I added the English name.  Not certain of what the correct localised names would be, and they're not on our Crowdin project either.  I experimented with a couple options for flags but I don't think they're really worth the effort to try to style inline in markdown and emoji-flags don't display as flags when someone's viewing via Windows.

File: README.md [hunk 1]
```diff
@@ -12,13 +12,27 @@ If you'd like to contribute, check past [our developer documentation](https://do
...
+We use [Crowdin](https://crowdin.com/project/xivanalysis) to manage and maintain our translations.
+If you would like to contribute new or corrected translation strings, please join our [CrowdIn project](https://crowdin.com/project/xivanalysis).
+
+We support the languages that FFXIV officially supports:
+
+- Chinese Simplified
+- French
+- German
+- Japanese
+- Korean
```
Comment (ayyaruq) [inline README.md:34]:
```suggestion
- Japanese (日本語)
- German (Deutsch)
- French (Français)
- Korean (한국어)
- Chinese Simplified (简体中文)
```

**Review by ayyaruq — CHANGES_REQUESTED** [review]
TIL emoji flags don't work in Windows, nice. Here's the localised strings for you.

**Review by ayyaruq — APPROVED** [review]
Noice (ナイス)

**Pull request merged**